wbsv/archiver.py

```diff
@@ -32,16 +34,14 @@ def archive(self, url: str) -> Union[Literal[False], Tuple[str, bool]]:
     def _try_savepagenow(wp: waybackpy.WaybackMachineSaveAPI) -> bool:
         """Error handler for saving with savepagenow."""
         try:
-            pass
             wp.save()
-        except waybackpy.exceptions.RedirectSaveError as e:
-            print(e, file=sys.stderr)
-            return False
-        except waybackpy.exceptions.WaybackError as e:
-            print(e, file=sys.stderr)
-            return False
-        except AttributeError as e:
-            print(e, file=sys.stderr)
+        except (
+            waybackpy.exceptions.RedirectSaveError,
+            waybackpy.exceptions.WaybackError,
+            requests.exceptions.TooManyRedirects,
+            AttributeError,
+        ) as e:
+            log_exception(e)
```

Comment:
```suggestion
            print(e, file=sys.stderr)
```

Reply:
I think it is useful to not call print directly so that it will be easier to replace it with the `logging` package later on which will allow the user to configure how the logs will be handled.

Right now all outputs are hardcoded to appear on stdout and stderr which is fine if wbsv is used as a standalone program (the user can redirect process outputs if needed) but when wbsv is used as a dependency there is no way to redirect outputs of wbsv.

Reply:
This project was originally implemented with only the CLI in mind, and was not intended to be used as a library. If you want to use it as a library, I strongly recommend Waybackpy.